pyproject.toml

```diff
@@ -118,5 +118,15 @@ checks = ["all", #report on all checks, except the following
 exclude = ['__init__' ] # don't report on objects that match any of these regex
 
 [tool.mypy]
+disable_error_code = ['type-arg', 'import-untyped', 'import-not-found', 'no-untyped-call']
+strict = true
+explicit_package_bases = true
 follow_imports = 'skip'
-exclude = ["tests", "docs"]
+exclude = ["tests", "docs" , "__init__.py"]
```

Comment:
I think you could remove this if you explicitly add the packages below.

Is there a reason we need to exclude `__init__.py` files?

Reply:
Nope, It was left over from when I was messing around.

Reply:
Also, 'tests' need to be still excluded because test folders are located within both the imap_processing and tools folders.
It's a give and take, you could make packages more defined and therefor won't need to use exclude or keep it more generic and include the exclude section. 
In this case I feel like keeping it generic is the better route.